packages/client/src/components/feed/LateralBar.tsx

```diff
@@ -13,9 +13,17 @@ import '../../styles/global.css';
 import '../../styles/home.css';
 import '../../styles/profile.css';
 import { useState } from 'react';
+const { graphql, useLazyLoadQuery } = require('react-relay');
 
 export default function LateralBar() {
   const [openNewTweetModal, setOpenTweetModal] = useState(false);
+  const { me } = useLazyLoadQuery(graphql`
```

Comment:
types? `useLazyLoadQuery<LateralBarQuery>()`

Reply:
nice catch, I'll fix it